test/TestMultiTag.cpp

```diff
@@ -171,6 +171,11 @@ void TestMultiTag::testReferences(){
     dt.addReference(da_2);
     CPPUNIT_ASSERT_MESSAGE(counterrmsg.str(), dt.referenceCount() == 2);
 
+    std::stringstream haserrmsg;
+    haserrmsg << "TestMultiTag::testReference: Has method did not work!";
```

Comment:
I'd maybe keep it consistent - having stream for now and then change it to CPPUNIT_ASSERT_MESSAGE("some message", x == y) everywhere, if it is worth so. But then maybe in the separate PR.